main.js

```diff
@@ -0,0 +1,240 @@
+    technology: ['html', 'Bootstrap', 'Ruby'],
+    languages: ['html', 'Bootstrap', 'Ruby on rails'],
+    // image: 'SVG/w6bgdt.png',
+    livelink: 'https://stevenmukama.github.io/Microverseportfolio/',
+    sourceLink: 'https://github.com/stevenmukama/Microverseportfolio',
+  },
+];
+
+const workcards = document.querySelector('.grids');
+
+projectCard.forEach((work, index) => {
+  workcards.innerHTML += ` <div id="id${index + 1}" class="cards">
+    <div class="carddetail" id="id${index + 1}carddetail">
+      <h2><span>${work.title}</span></h2>
+      <p>
+        ${work.description}
+      </p>
+      <ul class="cardlist">
+        <li class="button1" >${work.technology[0]}</li>
+        <li class="button1" >${work.technology[1]}</li>
+        <li class="button1" >${work.technology[2]}</li>
+      </ul>
+    </div>
+    <p class="card-liveLink" hidden>${work.livelink}</p>
+                <p class="card-sourceLink" hidden>${work.sourceLink}</p>
+                <p class="card-image" hidden>${work.image}</p>
+    <button class="button cardbtn" id="id${index + 1}button">See Project</button>
+  </div>`;
+});
+
+const modal = document.querySelector('.modal');
+
+const btn = document.querySelectorAll('.cardbtn');
+
+for (let i = 0; i < btn.length; i += 1) {
+  btn[i].addEventListener('click', () => {
+    const work = projectCard[i];
+    modal.style.display = 'block';
+
+    document.querySelector('.modal-top').innerHTML = `
+    <div class="modal-top">
+      <span class="project-title">
+        <span class="project-title">${work.popupdescription}</span>
+      </span>
+      `;
+
+    document.querySelector('.modal-info').innerHTML = `
+      <div class="project-tools">
+         <ul class="cardlist">
+        <li class="buttonlist" >${work.languages[0]}</li>
+        <li class="buttonlist" >${work.languages[1]}</li>
+        <li class="buttonlist" >${work.languages[2]}</li>
+      </ul>
+      </div>
+    </div>
+    <div class="modal-bottom" id="modal-bottom">
+      <div class="project-img">
+        <img src="images/Snapshootportfolio.png" alt=" Image" class="project-img-source">
+      </div>
+      <div class="project-details">
+        <div class="project-desc">
+        <p class="project-desc">${work.objectdescription} </p>
+        </div>
+
+        <div class="project-links">
+          <a href="" id="project-link-1" class="project-link-source">
+           See Source
+          <img src="/images/github.svg" alt="see-project-icon">
+        </a>
+        <a href="" id="project-link-1" class="project-link-live">
+        See live
+       <img src="/images/live.svg" alt="see-project-icon">
+     </a>
+        </div>
```

Comment:
- [ ] Please 🙏🏽 correct the path of the images, we could add a dot at the beginning `./images...` for a better user interface.

![image](https://user-images.githubusercontent.com/58642949/202827215-10e92eb5-52cc-493c-a33d-7c458d2cc8d2.png)